docs/community/tutorials-and-resources.md

```diff
@@ -101,12 +101,6 @@ Want your Django REST Framework talk/tutorial/article to be added to our website
 [django-rest-framework-course]: https://teamtreehouse.com/library/django-rest-framework
 [pycon-uk-2016]: https://www.youtube.com/watch?v=FjmiGh7OqVg
 [django-under-hood-2014]: https://www.youtube.com/watch?v=3cSsbe-tA0E
-[integrating-pandas-drf-and-bokeh]: https://machinalis.com/blog/pandas-django-rest-framework-bokeh/
```

Comment:
[integrating-pandas-drf-and-bokeh]: https://machinalis28.rssing.com/chan-63624552/latest.php#item5